integration-test-groups/foundation/language/src/main/java/org/apache/camel/quarkus/component/language/it/LanguageRoutes.java

```diff
@@ -16,15 +16,63 @@
  */
 package org.apache.camel.quarkus.component.language.it;
 
+import io.quarkus.runtime.annotations.RegisterForReflection;
+import jakarta.inject.Inject;
+import org.apache.camel.CamelContext;
 import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.util.Scanner;
 
+@RegisterForReflection(targets = { Scanner.class })
 public class LanguageRoutes extends RouteBuilder {
 
+    @Inject
+    CamelContext context;
+
     @Override
     public void configure() throws Exception {
+
+        /* Language routes using scripts passed to endpoints */
+
+        from("direct:languageBeanScript")
+                .to("language://bean:org.apache.camel.quarkus.component.language.it.CaseConverter::toUpper");
+        from("direct:languageConstantScript")
+                .to("language://constant:Hello from constant language script");
+        from("direct:languageExchangePropertyScript")
+                .setProperty("testProperty", simple("Hello ${body} from exchangeProperty language script"))
+                .to("language://exchangeProperty:testProperty");
+        from("file:target?fileName=test-file.txt&noop=true")
+                .to("language://file:File name is ${file:onlyname}")
+                .to("direct:languageFileOutput");
+        from("direct:languageHeaderScript")
+                .setHeader("testHeader", simple("Hello ${body} from header language script"))
+                .to("language://header:testHeader");
+        from("direct:languageHl7terserScript")
+                .to("language://hl7terser:PID-5")
+                .setBody(simple("Patient's surname is ${body}"));
+        from("direct:languageJsonPathScript")
+                .to("language://jsonpath:$.message");
+        from("direct:languageRefScript")
+                .to("language://ref:lowerCase");
         from("direct:languageSimpleScript")
                 .to("language://simple:Hello ${body} from simple language script");
+        from("direct:languageTokenizeScript")
+                .to("language://tokenize:,")
+                .setBody(simple("${body.next()}"));
+        from("direct:languageXpathScript")
+                .to("language://xpath:/message/text()");
+        from("direct:languageXqueryScript")
+                .to("language://xquery:upper-case(/message/text())");
+
+        /* Load simple scripts from resources */
+
         from("direct:languageSimpleResource")
-                .to("language://simple:resource:hello.simple.txt");
+                .to("language://simple:resource:hello.simple-res.txt");
+        from("direct:languageSimpleFile")
+                .to("language://simple:file:target/hello.simple-file.txt");
```

Comment:
Note sure this will work on quarkus-platform ? Or the test harness would need to create the file.

Wondering... this case should already covered in Camel test ? It there is nothing specific to Camel Quarkus, then it's fine to test it at Camel level only and it's shared with other runtime that way.

Reply:
I think it can't work on quarkus-platform and it needs to create a temporary file and copy the original content. We met the similar issue with `camel-quarkus-xstream` test with `file` schema.

Reply:
Or maybe it's the one created from LanguageTest.@BeforeAll